Cubical/Categories/LocallySmall/Displayed/Instances/Functor/Fibered.agda

```diff
@@ -0,0 +1,134 @@
+{-# OPTIONS --lossy-unification #-}
+module Cubical.Categories.LocallySmall.Displayed.Instances.Functor.Fibered where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Function
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.HLevels.More
+open import Cubical.Foundations.Isomorphism hiding (isIso)
+
+import Cubical.Data.Equality as Eq
+open import Cubical.Data.Unit
+open import Cubical.Data.Sigma
+open import Cubical.Data.Sigma.More
+
+open import Cubical.Reflection.RecordEquiv.More
+
+import Cubical.Categories.Category as Small
+open import Cubical.Categories.LocallySmall.Category.Base
+open import Cubical.Categories.LocallySmall.Category.Small
+open import Cubical.Categories.LocallySmall.Instances.Unit
+open import Cubical.Categories.LocallySmall.Instances.Functor.Fibered
+open import Cubical.Categories.LocallySmall.Functor.Base
+open import Cubical.Categories.LocallySmall.NaturalTransformation.Fibered
+open import Cubical.Categories.LocallySmall.Instances.Indiscrete
+open import Cubical.Categories.LocallySmall.Variables
+open import Cubical.Categories.LocallySmall.Constructions.ChangeOfObjects
+
+open import Cubical.Categories.LocallySmall.Displayed.Category
+open import Cubical.Categories.LocallySmall.Displayed.Functor.Base
+open import Cubical.Categories.LocallySmall.Displayed.Constructions.Weaken
+open import Cubical.Categories.LocallySmall.Displayed.Constructions.ChangeOfObjects
+open import Cubical.Categories.LocallySmall.Displayed.Constructions.Reindex.Base
+open import Cubical.Categories.LocallySmall.Displayed.Constructions.Reindex.Properties
+open import Cubical.Categories.LocallySmall.Displayed.Constructions.BinProduct.Base
+open import Cubical.Categories.LocallySmall.Displayed.NaturalTransformation.Fibered.Base
+
+open Category
+open Categoryᴰ
+
+open Functorᴰ
+open Liftω
+open Σω
+
+module FiberedFunctorCategoryᴰ
+  {C : SmallCategory ℓC ℓC'}
+  {ℓCᴰ ℓCᴰ'}
+  (Cᴰ : SmallCategoryᴰ C ℓCᴰ ℓCᴰ')
+  {D : Category Dob DHom-ℓ}
+  {Dobᴰ DHom-ℓᴰ}
+  (Dᴰ : GloballySmallCategoryᴰ D Dobᴰ DHom-ℓᴰ)
+  {Eob-ℓᴰ Eobᴰ EHom-ℓᴰ}
+  (Eᴰ : SmallFibersCategoryᴰ D Eob-ℓᴰ Eobᴰ EHom-ℓᴰ)
+  {Dᴰᴰ-ℓ Dobᴰᴰ DHom-ℓᴰᴰ}
+  (Dᴰᴰ : SmallFibersᴰCategoryᴰ Dᴰ Eᴰ Dᴰᴰ-ℓ Dobᴰᴰ DHom-ℓᴰᴰ)
+  where
+  open FibNatTransᴰDefs Cᴰ Dᴰ Eᴰ Dᴰᴰ public
+  open FibNatTransᴰ
+  private
+    module C = SmallCategory C
+    module Cᴰ = SmallCategoryᴰ Cᴰ
+    module D = CategoryNotation D
+    module Dᴰ = CategoryᴰNotation Dᴰ
+    module Eᴰ = CategoryᴰNotation Eᴰ
+    module Dᴰᴰ = CategoryᴰNotation Dᴰᴰ
+
+    C⇒Eᴰ : Categoryᴰ D FiberedFunctor _
+    C⇒Eᴰ = FIBERED-FUNCTOR C Eᴰ
+
+    Dᴰ×C⇒Eᴰ = Dᴰ ×ᴰ C⇒Eᴰ
+    Dᴰ×Eᴰ = Dᴰ ×ᴰ Eᴰ
+    module Dᴰ×C⇒Eᴰ = CategoryᴰNotation Dᴰ×C⇒Eᴰ
+    module Dᴰ×Eᴰ = CategoryᴰNotation Dᴰ×Eᴰ
+
+  FIBERED-FUNCTORᴰ :
```

Comment:
Should we just change `FiberedFunctor(ᴰ)` to `Functor(ᴰ)`? https://ncatlab.org/nlab/show/fibred+functor means a displayed functor that preserves cartesian lifts.